cliparser.py

```diff
@@ -26,7 +26,7 @@
 
 
 class CliParser(argparse.ArgumentParser):
-    def __init__(self, cli_support=None, board_names=None, add_help=True):
+    def __init__(self, *, cli_support=None, board_names=None, add_help=True, stack_name=None):
```

Comment:
Why do we need this `*`?

Reply:
[undone](https://github.com/intel/auto-pts/pull/742/commits/f161ae7604b41f8fdaa7809d10c0ece69ae6f469)